Providers/Modules/Plugins/ChangeTracking/conf/change_tracking_inventory.mof

```diff
@@ -0,0 +1,38 @@
+/*
+@TargetNode='Localhost'
+*/
+
+instance of MSFT_nxPackageResource
+{
+                Name = "*";
+                ResourceId = "[MSFT_nxPackageResource]Inventory";
```

Comment:
Your indentation looks "weird" here, particularly in relation to OMI_ConfigurationDocument below. Isn't one tab sufficient?